BogdanSokhatiuk/src/main/java/jdbcmySql/model/Constants.java

```diff
@@ -0,0 +1,17 @@
+package jdbcmySql.model;
+
+/**
+ * Created by lost on 19.11.2016.
+ */
+public class Constants {
+    public static final String SQLSELECT1 = "Select * from students";
```

Comment:
set appropriate and clear names to constants

i recommend to group queries by a class name(model)